source/docs/software/hardware-apis/pneumatics/pneumatics.rst

```diff
@@ -6,89 +6,101 @@ Operating pneumatic cylinders
 
 .. tabs::
 
     .. code-tab:: java
 
-        Compressor c = new Compressor(0, PneumaticsModuleType.CTREPCM);
+        Compressor pcm = new Compressor(0, PneumaticsModuleType.CTREPCM);
+        Compressor pch = new Compressor(1, PneumaticsModuleType.REVPH);
 
-        c.setClosedLoopControl(true);
-        c.setClosedLoopControl(false);
+        pcm.enableDigital();
+        pcm.disable();
 
-        boolean enabled = c.enabled();
-        boolean pressureSwitch = c.getPressureSwitchValue();
-        double current = c.getCompressorCurrent();
+        boolean enabled = pcm.enabled();
+        boolean pressureSwitch = pcm.getPressureSwitchValue();
+        double current = pcm.getCompressorCurrent();
 
     .. code-tab:: c++
 
-        frc::Compressor c{0, frc::PneumaticsModuleType::CTREPCM};
+        frc::Compressor pcm{0, frc::PneumaticsModuleType::CTREPCM};
+        frc::Compressor pch{1, frc::PneumaticsModuleType::REVPH};
 
-        c.SetClosedLoopControl(true);
-        c.SetClosedLoopControl(false);
-
-        bool enabled = c.Enabled();
-        bool pressureSwitch = c.GetPressureSwitchValue();
-        double current = c.GetCompressorCurrent();
+        pcm.EnableDigital();
+        pcm.Disable();
 
+        bool enabled = pcm.Enabled();
+        bool pressureSwitch = pcm.GetPressureSwitchValue();
+        double current = pcm.GetCompressorCurrent();
 
 
+The Pneumatic Hub also has methods for enabling compressor control using the REV Analag Pressure Sensor (``enableAnalog`` method).
 
 Solenoid control
 ----------------
 
-FRC teams use solenoids to perform a variety of tasks, from shifting gearboxes to operating robot mechanisms. A solenoid is a valve used to electronically switch a pressurized air line "on" or "off". For more information on solenoids, see `this wikipedia article <https://en.wikipedia.org/wiki/Solenoid_valve>`__. Solenoids are controlled by a robot's Pneumatics Control Module, or PCM, which is in turn connected to the robot's roboRIO via CAN. The easiest way to see a solenoid's state is via the small red LED (which indicates if the valve is "on" or not), and solenoids can be manually actuated when un-powered with the small button adjacent to the LED.
+FRC teams use solenoids to perform a variety of tasks, from shifting gearboxes to operating robot mechanisms. A solenoid is a valve used to electronically switch a pressurized air line "on" or "off". For more information on solenoids, see `this wikipedia article <https://en.wikipedia.org/wiki/Solenoid_valve>`__. Solenoids are controlled by a robot's Pneumatics Control Module, or Pneumatic Hub, which is in turn connected to the robot's roboRIO via CAN. The easiest way to see a solenoid's state is via the LEDs on the PCM or PCH (which indicates if the valve is "on" or not), and solenoids can be manually actuated when un-powered with the small button adjacent to the LED.
```

Comment:
```suggestion
FRC teams use solenoids to perform a variety of tasks, from shifting gearboxes to operating robot mechanisms. A solenoid is a valve used to electronically switch a pressurized air line "on" or "off". For more information on solenoids, see `this wikipedia article <https://en.wikipedia.org/wiki/Solenoid_valve>`__. Solenoids are controlled by a robot's Pneumatics Control Module, or Pneumatic Hub, which is in turn connected to the robot's roboRIO via CAN. The easiest way to see a solenoid's state is via the LEDs on the PCM or PH (which indicates if the valve is "on" or not), and solenoids can be manually actuated when un-powered with the small button adjacent to the LED.
```

Reply:
This change makes it sound like you're saying that the small button is next to the LED on the PH/PCM, not the LED on the solenoid itself.